### ISSUE TITLE
Redesign a index.html 

### ISSUE THREAD

**Issue by samrat2k03** [thecuriousteam/PyLibLog#83]
i want to redesign the index.html and other pages too ...

**Comment by thecuriousteam** [comment]
Yes You can do it, once done please attach your screenshot and please join the onboarding group

Developers needs to onboard to opensource community for easy information flow along with admins, please join with this link  [Curious Developers Community - Hacktoberfest ](https://chat.whatsapp.com/HNu3VaLPK3z3yt3JFUs7D7)</br>

Standrad Website Design reference
      1. https://dart-tutorial.com/
      2. https://www.w3schools.com/html/
      3. https://www.studytonight.com/html5-references/html-a-tag

**Comment by samrat2k03** [comment]
> Yes You can do it, once done please attach your screenshot and please join the onboarding group
> 
> Developers needs to onboard to opensource community for easy information flow along with admins, please join with this link [Curious Developers Community - Hacktoberfest ](https://chat.whatsapp.com/HNu3VaLPK3z3yt3JFUs7D7)
> 
> Standrad Website Design reference 1. https://dart-tutorial.com/ 2. https://www.w3schools.com/html/ 3. https://www.studytonight.com/html5-references/html-a-tag

sure 👍

**Comment by samrat2k03** [comment]
i think we need to move on javaScript framework?  (reactJs)

**Comment by thecuriousteam** [comment]
Yes


Thank You

## Instructions

1. Yes you can start working on it by taking latest pull from main branch
2. Don't Forget to join onboarding group for knowledge flow and direct conversation and support from other developers link  [Curious Developers Community - Hacktoberfest ](https://chat.whatsapp.com/HNu3VaLPK3z3yt3JFUs7D7)
3. Please attach the screenshots and video of the development work once it is done
4. Contributions more than 3 will be awarded a certificate of appreciation from curious ecosystem please mail to (admin@curiousdevelopers.in) for claiming certificate and please attach your issue number 
5. Respect Co Developers And Please Reach out to admins if any issues

Thanks & Regards
The Curious Team
